src/content/docs/microsoft/Azure/azure-devops-services.md

```diff
@@ -0,0 +1,79 @@
+- **Azure Repos:** Azure Repos help to keep track of the coding work and makes it possible for a team to work on the same project.
+- **Azure Pipelines:** Azure Pipelines is a tool that helps automate the process of getting the project ready and ensuring it works properly. It is a tool for build and testing.
+- **Azure Test Plans:** Azure Test Plans helps teams make sure the project works well. It's used right in the web browser and gives you everything you need to carry out different types of testing.
+- **Azure Artifacts:** Azure Artifacts is a tool that lets us handle all code and packages in one place. It makes it easy to share the code that is written with other people on the team, company and more.
+
+### Azure Boards
+
+In this study, I will concentrate on Azure Boards, a core feature of Azure DevOps that facilitates project management and collaboration.
+
+## Market Comparison
+
+### Advantages
+
+Using Azure Boards comes with a lot of advantages. The platform's visual and interactive tools, like Kanban boards, make it easy for teams to see progress and plan work effectively. It is easy to collaborate on issues and tasks with the comment section. If you want to get someone's attention or include them in the conversation, you just type "@" followed by their name.
+
+Azure Boards lets you keep a lot of information about your projects. You can add detailed descriptions, pictures right in the text, and big files up to 60 MB to your tasks. It's also simple to connect related tasks together, which helps you keep everything organized and track how tasks change over time.
+
+In Azure Boards, it's easy to find the things you need, even if your project is really big. They have good search tools and you can set up notifications the way you like. Keeping an eye on how your project is doing is simple too, with dashboards and tools. You can make your own reports and see the big picture of your project with these customizable features.
+
+### Disadvantages
+
+Azure Boards is compatible with the Scaled Agile Framework (SAFe), although it isn't specifically created for any large-scale Agile framework. To handle budgeting and reporting for Agile at a larger scale, it needs more setup and adjustments because it doesn't come with a lot of these features ready out of the box.
+
+### Comparison to GitHub Projects
+
+GitHub Projects and Azure Boards are both project management tools that offer issue and progress tracking for software development. It is a tool you can use to organize issues. You can sort your tasks, make charts to see how you're doing, and note down extra details your team might need. It's made to be easy to change, so it can match the way your team likes to work.
+
+It works seamlessly with GitHub repositories, issues, and pull requests, since it is a part of the Github platform. It's designed to be flexible, and thats allowing teams to use it without enforcing a particular methodology.
```

Comment:
```suggestion
It works seamlessly with GitHub repositories, issues, and pull requests, since it is a part of the Github platform. It's designed to be flexible, allowing teams to use it without enforcing a particular methodology.
```